DESIGN_DOCS.md

```diff
@@ -0,0 +1,65 @@
+# Design Documents
+
+## Purposes of this document
+* To describe what our design documents are for, their structure, and what goes in them.
+* To describe how DSNP Design Documents are produced and where they're published.
+
+## What Is a Design Document?
+It's a document clearly describing a problem, a solution to that problem, and why we think the proposal is the right solution.
+It has enough information to begin writing stories for implementing the solution in the document.
```

Comment:
You made some good comments earlier on process that you might pull in here, just for more clarity on how this is supposed to work.  e.g. The ideal process is: create the design doc --> discuss, review and merge design doc --> create issue in the respective GitHub project (e.g. a story) --> code implementation.